norns/clocks.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+parent: scripting
+grand_parent: norns
+title: clocks
+nav_order: 3
+permalink: /norns/clocks/
+---
+
+# clocks
+
+the clock library provides a way to create timed function calls: loops, repetition, and delays. synchronization is possible and time base can come from a variety of sources.
+
+## commands
+
+- `id = clock.run( func )` &mdash; start a new clock with function _func_. returns _id_
+- `clock.cancel( id )` &mdash; cancel clock _id_ 
+- `clock.sleep( time )` &mdash; resume in _time_ seconds
+- `clock.sync( beats )` &mdash; resume in _beats_ according to tempo
```

Comment:
sync doesn't exactly resume _in_ `beats` beats, but rather when `beats` beats are reached on the timeline, e.g. if you call `clock.sync(3)` around beat 2 according to the clock, coroutine will wake up in 1 beat.

upd: i see that it's covered below, so is probably fine to keep here as is.

Reply:
i think this could be clarified -- it does sound like `clock.sync(3)` would resume in 3 beats from whatever beat its called, rather than resume on beat 3 according to the global clock.